parm/archive/gdas.yaml.j2

```
@@ -70,6 +79,10 @@ gdas:
             {% endif %}
             {% if DO_AERO_ANL %}
         - "{{ COMIN_CHEM_ANALYSIS | relpath(ROTDIR) }}/{{ head }}aerostat.tgz"
+        - "{{ COMIN_CONF | relpath(ROTDIR) }}/{{ head }}aerovar.yaml"
+                {% for itile in range(1,7) %}
+        - "{{ COMIN_CHEM_ANALYSIS | relpath(ROTDIR) }}/aeroinc.{{ cycle_YMD }}.{{ cycle_HH }}0000.fv_tracer.res.tile{{ itile }}.nc"
```

This works but if you want to make it cleaner, here is a suggestion:
```suggestion
        - "{{ COMIN_CHEM_ANALYSIS | relpath(ROTDIR) }}/aeroinc.{{ current_cycle | strftime('%Y%m%d.%H0000') }}.fv_tracer.res.tile{{ itile }}.nc"
```

---

probably not worth it